### PR TITLE
LRDOCS-8009 Sidecar in WebSphere and WebLogic

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-8009 Forwarding with no changes. @rbohl @jhinkey @arboliveira

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-weblogic.md [hunk 1]
```diff
@@ -121,6 +122,22 @@ DXP communicates with your database via JDBC. Add your database JDBC driver JAR
 
 Please see the [compatibility matrix](https://www.liferay.com/documents/10182/246659966/Liferay+DXP+7.2+Compatibility+Matrix.pdf/ed234765-db47-c4ad-7c82-2acb4c73b0f9) for a list of supported databases.
 
+## Install Elasticsearch Archives
+
+When you start Liferay DXP 7.3, it installs and starts a default [sidecar](../../../using-search/installing-and-upgrading-a-search-engine/elasticsearch/using-the-built-in-elasticsearch.md) Elasticsearch server. For the installation to succeed, you must provide some archives:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-weblogic.md:127]: FYI @rbohl @sez11a this link is broken - the `using-the-built-in-elastic-search` article has not been ported yet.

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-websphere.md [hunk 2]
```diff
@@ -150,6 +151,22 @@ DXP communicates with your database via JDBC. Add your database JDBC driver JAR
 
 Note that although a Hypersonic database is bundled with DXP and is fine for testing purposes, do not use it for production DXP instances.
 
+## Installing Elasticsearch Archives
+
+When you start Liferay DXP 7.3, it installs and starts a default [sidecar](../../../using-search/installing-and-upgrading-a-search-engine/elasticsearch/using-the-built-in-elasticsearch.md) Elasticsearch server. For the installation to succeed, you must provide some archives:
```
Comment (jrhoun) [inline docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-websphere.md:156]: FYI @rbohl @sez11a this link is broken - the using-the-built-in-elastic-search article has not been ported yet.

**Comment by jrhoun** [comment]
Merged. Please see my comments though -- hopefully those related articles come along soon!

File: docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-websphere.md (hunk 2 above)
Comment (rbohl) [reply docs/dxp/7.x/en/installation-and-upgrades/installing-liferay/installing-liferay-on-an-application-server/installing-on-websphere.md:156]: Not a port. This is new documentation that's in SME review right now.

**Pull request closed without merging**